.pkgr.yml

```diff
@@ -7,10 +7,15 @@ targets:
       - git
   debian-8:
     <<: *debian
+  ubuntu-12.04:
+    <<: *debian
   ubuntu-14.04:
     <<: *debian
-  ubuntu-12.04:
+  ubuntu-16.04:
     <<: *debian
+    build_dependencies:
```

Comment:
Why were these added? and why not add them to `debian-7` instead as I'm guessing all builds would need them?